scrapy/utils/conf.py

```diff
@@ -34,6 +34,16 @@ def _map_keys(compdict):
             _check_components(compdict)
             return {convert(k): v for k, v in six.iteritems(compdict)}
 
+    def _ensure_numeric_value(compdict):
+        for name, value in six.iteritems(compdict):
+            if value is None or isinstance(value, (int, float)):
+                continue
+            try:
+                compdict[name] = float(value)
```

Comment:
I'll drop support for strings then 😄
Out of curiosity, what do you mean by "order will change, as compared to scrapy 1.3"?